python/pyproject_rocm.toml

```diff
@@ -0,0 +1,178 @@
+  "tiktoken",
+  "torch_memory_saver==0.0.9rc2",
+  "torchao==0.9.0",
+  "tqdm",
+  "transformers==4.57.1",
+  "uvicorn",
+  "uvloop==0.21.0",
+  "grpcio==1.75.1", # keep it align with compile_proto.py
+  "grpcio-tools==1.75.1", # keep it align with compile_proto.py
+  "grpcio-reflection==1.75.1", # required by srt/entrypoints/grpc_server.py
+]
+
+[project.optional-dependencies]
+decord = ["decord2"]
+test = [
+  "accelerate",
+  "expecttest",
+  "gguf",
+  "jsonlines",
+  "matplotlib",
+  "pandas",
+  "peft",
+  "pytest",
+  "sentence_transformers",
+  "tabulate",
+]
+tracing = [
+  "opentelemetry-api",
+  "opentelemetry-exporter-otlp",
+  "opentelemetry-exporter-otlp-proto-grpc",
+  "opentelemetry-sdk",
+]
+
+rocm640 = [
+    "torch @ https://download.pytorch.org/whl/rocm6.4/torch-2.9.0%2Brocm6.4-cp310-cp310-manylinux_2_28_x86_64.whl ; python_version == '3.10'",
+    "torch @ https://download.pytorch.org/whl/rocm6.4/torch-2.9.0%2Brocm6.4-cp311-cp311-manylinux_2_28_x86_64.whl ; python_version == '3.11'",
+    "torch @ https://download.pytorch.org/whl/rocm6.4/torch-2.9.0%2Brocm6.4-cp312-cp312-manylinux_2_28_x86_64.whl ; python_version == '3.12'",
+    "torch @ https://download.pytorch.org/whl/rocm6.4/torch-2.9.0%2Brocm6.4-cp313-cp313-manylinux_2_28_x86_64.whl ; python_version == '3.13'",
+    "torch @ https://download.pytorch.org/whl/rocm6.4/torch-2.9.0%2Brocm6.4-cp314-cp314-manylinux_2_28_x86_64.whl ; python_version == '3.14'",
+    "torchvision @ https://download.pytorch.org/whl/rocm6.4/torchvision-0.24.0%2Brocm6.4-cp310-cp310-manylinux_2_28_x86_64.whl ; python_version == '3.10'",
+    "torchvision @ https://download.pytorch.org/whl/rocm6.4/torchvision-0.24.0%2Brocm6.4-cp311-cp311-manylinux_2_28_x86_64.whl ; python_version == '3.11'",
+    "torchvision @ https://download.pytorch.org/whl/rocm6.4/torchvision-0.24.0%2Brocm6.4-cp312-cp312-manylinux_2_28_x86_64.whl ; python_version == '3.12'",
+    "torchvision @ https://download.pytorch.org/whl/rocm6.4/torchvision-0.24.0%2Brocm6.4-cp313-cp313-manylinux_2_28_x86_64.whl ; python_version == '3.13'",
+    "torchvision @ https://download.pytorch.org/whl/rocm6.4/torchvision-0.24.0%2Brocm6.4-cp314-cp314-manylinux_2_28_x86_64.whl ; python_version == '3.14'",
+    "pytorch-triton-rocm @ https://download.pytorch.org/whl/pytorch_triton_rocm-3.5.0-cp310-cp310-linux_x86_64.whl ; python_version == '3.10'",
+    "pytorch-triton-rocm @ https://download.pytorch.org/whl/pytorch_triton_rocm-3.5.0-cp311-cp311-linux_x86_64.whl ; python_version == '3.11'",
+    "pytorch-triton-rocm @ https://download.pytorch.org/whl/pytorch_triton_rocm-3.5.0-cp312-cp312-linux_x86_64.whl ; python_version == '3.12'",
+    "pytorch-triton-rocm @ https://download.pytorch.org/whl/pytorch_triton_rocm-3.5.0-cp313-cp313-linux_x86_64.whl ; python_version == '3.13'",
+    "pytorch-triton-rocm @ https://download.pytorch.org/whl/pytorch_triton_rocm-3.5.0-cp314-cp314-linux_x86_64.whl ; python_version == '3.14'",
+    "sgl-kernel @ <URL TODO>",
```

Comment:
pyproject_rocm.toml still uses <URL TODO>, so the ROCm pip install flow cannot run end-to-end.  I am wondering how this was tested?